### DISCUSSION TITLE
Manage profile page button disabled

### DISCUSSION THREAD

**Discussion by MaorBemdoo** [EddieHubCommunity/LinkFree#10279] [Q&A]
Why is the Button Component on the manage profile page of a user disabled as a premium member

**Comment by Vijaykv5** [comment]
Could you be more specific @MaorBemdoo ?

**Reply by MaorBemdoo** [reply]
https://github.com/EddieHubCommunity/BioDrop/assets/100993978/aed58d65-8ad3-414a-8a42-1158235b00ab

I guess this video should explain better

**Reply by Vijaykv5** [reply]
@eddiejaoude could you check on this & possible reason to occur for?
It exists for me also!
If so, we can move this discussion onto an issue.

**Reply by eddiejaoude** [reply]
Thank you both for the collaboration. Yes let us know more an issue, as it might be a bug

**Reply by eddiejaoude** [reply]
Please  @MaorBemdoo can you create an issue (bug) and include all the information and video

Thank you @Vijaykv5 for verifying and the tag 👍

**Reply by MaorBemdoo** [reply]
I have created an issue: #10291

**Reply by MaorBemdoo** [reply]
🙏Thank you @eddiejaoude  and @Vijaykv5 for responding to my question
